addons/arsenal/CfgACEArsenalStats.hpp

```diff
@@ -0,0 +1,96 @@
+class CfgACEArsenalStats {
```

Comment:
Should be renamed to `class ACE_Arsenal_Stats` (`GVAR(Stats)`) to fit other configs we have (`ACE_detector`, `ACE_UI`, `ACE_Settings`, `ACE_Extensions`, etc.).